src/components/courseSettings/DownloadPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardHeader, CardTitle, CardBody, Table } from 'reactstrap'
+import { withBaseUrl } from '../../util'
+
+interface DownloadPanelProps {
+  course: {
+    id: number
+  }
+}
+
+const DownloadPanel = ({ course }: DownloadPanelProps) => {
+  return (
+    <Card className="mb-3">
+      <CardHeader>
+        <CardTitle tag="h5" className="mb-0">
+          Downloads
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
```

Comment:
The table itself should provide suitable padding, you can likely remove the wrapping `CardBody` for a better design.